salt/metalk8s/orchestrate/upgrade/etcd.sls

```diff
@@ -0,0 +1,57 @@
+{%- set dest_version = pillar.orchestrate.dest_version %}
+{%- set etcd_nodes = salt.metalk8s.minions_by_role('etcd') %}
+{%- set control_plane_ip = salt.saltutil.runner(
+    'mine.get',
+    tgt=pillar.bootstrap_id,
+    fun='control_plane_ip'
+)[pillar.bootstrap_id] %}
+
+{%- set pillar_data = {
+    'repo': {
+        'host': control_plane_ip
+    },
+    'registry_ip': control_plane_ip
+} %}
+
+Check etcd cluster health:
+  module.run:
+    - metalk8s.check_etcd_health:
+      - hostname: {{ pillar.bootstrap_id }}
```

Comment:
Since this is a `runner`, which by definition runs on the Salt master, which is the 'bootstrap' node, can we replace this with the local hostname instead?

Reply:
Hmm I'm not really like because strictly it's not the hostname of the salt-master that we want. We want an hostname of one node with etcd.

Reply:
Ugh. Really need to use the etcd Python API, then run this stuff *from the master*. Could you refer to this one in the relevant ticket as well?

Reply:
Done